src/backend/joanie/tests/core/models/test_quote.py

```diff
@@ -49,3 +59,92 @@ def test_models_quote_context_should_be_generated_before_organization_signed_on(
             "You must generate the quote context before signing the quote."
             in str(context.exception)
         )
+
+    def test_models_quote_reference_uniqueness(self):
```

Comment:
For this test, we should override `JOANIE_QUOTE_REFERENCE_PREFIX` setting with something else than the default value, and maybe add a variable containing `f"{settings.JOANIE_QUOTE_REFERENCE_PREFIX}_{year}_"` ?